test/parallel/test-fs-stream-no-events-after-destroy.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const fs = require('fs');
+const assert = require('assert');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+const test = (stream) => {
+  const err = new Error('DESTROYED');
+  stream.destroy(err);
+  stream.on('open', common.mustNotCall());
+  stream.on('ready', common.mustNotCall());
+  stream.on('close', common.mustCall());
+  stream.on('error', common.mustCall((er) => {
+    assert.strictEqual(err, er);
+  }));
```

Comment:
Can you also test the order of these? `'error'` should come after `'close'`.

Reply:
Ouch! My bad, I got confused.. error should come before close, not after. (I’m sorry for this).

The previous behavior was correct, it was just needed to update the tests to check it